packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -304,12 +340,40 @@ export default function OrgNav({
           </OrgNavLink>
         )}
 
-        <UserMenu
-          account={account}
-          activeProduct={activeProduct}
-          urls={urls}
-          hosts={hosts}
-        />
+        {isOnPrem ? (
+          <div className={onPremMenuWrapper}>
+            <UserMenuTrigger
+              name={account.firstName}
+              open={onPremMenuOpen}
+              setOpen={setOnPremMenuOpen}
+            />
+            <Menu open={onPremMenuOpen} setOpen={setOnPremMenuOpen}>
+              <MenuItem href={urls.onPrem.profile}>Profile</MenuItem>
+              <MenuItem href={urls.onPrem.mfa}>
+                Two-factor Authentication
+              </MenuItem>
+              <MenuItem href={urls.onPrem.personalization}>
+                Personalization
+              </MenuItem>
+              <MenuItem href={urls.onPrem.invitations}>Invitations</MenuItem>
+              <MenuItem href={urls.onPrem.organizations}>
+                Organizations
+              </MenuItem>
+              <MenuItem href={urls.onPrem.featureRequest}>
```

Comment:
Right now, I think you have the default for this set to an empty string. Is there a corresponding page for this? If not, maybe we can conditionally hide this for now?

Reply:
I asked in our channel if people knew where this should point to. I do not. If you and @hswolff don't know yet, then I can definitely hide for now.